### PR TITLE
[KV]: Add methods to load/save bytes for MinIOKV

### PR DESCRIPTION
**Pull request by yah01**
Signed-off-by: yah01 <yang.cen@zilliz.com>
#10468

### CODE REVIEW

File: internal/kv/minio/minio_kv.go [hunk 1]
```diff
@@ -187,6 +233,23 @@ func (kv *MinIOKV) MultiLoad(keys []string) ([]string, error) {
...
+func (kv *MinIOKV) MultiLoadBytes(keys []string) ([][]byte, error) {
+	var resultErr error
+	objectsValues := make([][]byte, 0, len(keys))
+
+	for _, key := range keys {
+		objectValue, err := kv.LoadBytes(key)
+		if err != nil {
+			if resultErr == nil {
+				resultErr = err
+			}
```
Comment (congqixia) [inline internal/kv/minio/minio_kv.go:245]: Current `MultiLoadBytes` fails if any key `LoadBytes` fails. May just return here for shortcut.

Comment (yah01) [reply internal/kv/minio/minio_kv.go:245]: Then what about `MultiLoad()`? Make them both return when encounter the first error?

Comment (czs007) [reply internal/kv/minio/minio_kv.go:245]: > 

lgtm

**Comment by yah01** [comment]
rerun ut

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**